packages/jest-snapshot/src/index.ts

```diff
@@ -4,17 +4,16 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {HasteFS} from 'types/HasteMap';
-import type {MatcherState} from 'types/Matchers';
-import type {Path, SnapshotUpdateState} from 'types/Config';
-import type {SnapshotResolver} from 'types/SnapshotResolver';
-
 import fs from 'fs';
+import {Config} from '@jest/types';
+import {FS as HasteFS} from 'jest-haste-map'; // eslint-disable-line import/no-extraneous-dependencies
+
 import diff from 'jest-diff';
 import {EXPECTED_COLOR, matcherHint, RECEIVED_COLOR} from 'jest-matcher-utils';
+import {MatcherState} from 'expect';
```

Comment:
an you put this in `@jest/types` instead? We'll move it back into `expect` when it is migrated to TS (or probably keep it in `@jest/types` until the next major since `expect` doesn't use ESM)

Reply:
/cc @natealcedo relevant for your `expect` migration 🙂